pkgs/by-name/co/color-scheme-generator/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+  openssl,
+  sqlite,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "color-scheme-generator";
+  version = "0.9.2.1";
+
+  src = fetchFromGitHub {
+    owner = "nikolaizombie1";
+    repo = "color_scheme_generator";
+    tag = "v${version}";
+    hash = "sha256-tSoM6MP58vOoKRMQjNq8nKMiVTfS9N/mckchAPcUM0g=";
+  };
+
+  cargoHash = "sha256-A8ld2c8ZcqaZN29Fz085FQy8UpugIDMOKT01r+SfCcc=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    openssl
+    sqlite
+  ];
+
+  env = {
+    OPENSSL_NO_VENDOR = 1;
+  };
+
```

Comment:
`passthru.updateScript = gitUpdater { rev-prefix = "v"; };`

Update script so the bot sends the updates as pull requests, and, because you are the maintainer, you can merge those PRs tagging another bot.